packages/loader/container-loader/src/container.ts

```diff
@@ -126,6 +127,11 @@ export interface IContainerLoadOptions {
      * Loads the Container in paused state if true, unpaused otherwise.
      */
     pause?: boolean;
+
+    /**
+     * Max time container will wait for a leave message of a disconnected client.
```

Comment:
Can you please clarify (in comments) if it's ms or seconds

Reply:
Added.